docs/operation/operation.md

```diff
@@ -926,6 +931,7 @@ The parameters of `-passive-health-check` option are:
 + `min-drop-probabilty=<0.0 <= p <= 1.0>` - the minimum possible probability of unhealthy endpoint being not considered while choosing the endpoint for the given request. The same value is in fact used as minimal failed requests ratio for PHC to be enabled for this endpoint
 + `max-drop-probabilty=<0.0 <= p <= 1.0>` - the maximum possible probability of unhealthy endpoint being not considered
```

Comment:
I think we also miss here that `min-drop-probabilty < max-drop-probabilty` should hold true. Not sure if we check this in code somewhere but sounds like we should.